src/toggle-switch/toggle-switch.scss

```diff
@@ -141,10 +159,19 @@
     cursor: not-allowed;
     background-color: var(--iui-color-background-disabled);
     border-color: var(--iui-color-background-disabled);
+    @media (forced-colors: active) {
+      background-color: GrayText;
+      border-color: GrayText;
+    }
 
     &::after {
       background-color: var(--iui-color-foreground-body);
       opacity: var(--iui-opacity-5);
+
+      @media (forced-colors: active) {
+        background-color: Canvas;
```

Comment:
Should we try to use different styling for checked/unchecked when in disabled state?

![image](https://user-images.githubusercontent.com/9084735/163839126-54ea1b82-1109-4642-8de0-e2cdd2b062c5.png)


vs

![image](https://user-images.githubusercontent.com/9084735/163838993-33f5774e-0eee-4605-97b8-9be42b5baca3.png)

Reply:
I support this, change made:
![Screen Shot 2022-04-25 at 10 41 58 AM](https://user-images.githubusercontent.com/849817/165112767-74fc48d5-0916-4caa-98e9-64070f39704a.png)
![Screen Shot 2022-04-25 at 10 42 30 AM](https://user-images.githubusercontent.com/849817/165112878-bd2c9e26-0147-45db-99cc-42b98526143a.png)

Reply:
Looks great. This is from Windows 11 with more realistic colors:
![image](https://user-images.githubusercontent.com/9084735/165139668-5dbf34f9-620e-4796-9043-b1a3d5a52eaa.png)